components/application/navigation.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import { Icon } from '@oacore/design'
+
+import styles from './navigation.css'
+
+import { Drawer } from 'design'
+
+const Navigation = ({ children, ...passProps }) => (
+  <Drawer.List {...passProps}>{children}</Drawer.List>
+)
+
+const NavigationItem = ({ children, href }) => (
+  <Drawer.Item href={href}>{children}</Drawer.Item>
+)
+
+Navigation.Item = NavigationItem
+
+const titleMap = {
+  overview: 'Overview',
+  data: 'Data',
+  statistics: 'Statistics',
+  plugins: 'Plugins',
+  settings: 'Settings',
+}
+
+const captions = Object.fromEntries(
+  ['overview', 'data', 'statistics', 'plugins', 'settings'].map(activity => [
+    activity,
+    <>
+      <Icon
+        className={styles.itemIcon}
+        src={`/design/icons.svg#${activity}`}
+        alt
```

Comment:
Is it good to pass empty alt?

Reply:
Yes if you do not want text to be read (I guess).

Reply:
Then why passing alt at all?

Reply:
`alt` is required by the [standard](https://html.spec.whatwg.org/multipage/images.html#alt).

In addition I found that it should not be empty but can be in such cases. In this case I do not want it to be read, so I pass `aria-hidden` below and I do not care if the image is not displayed because the button has a label for it. It's only styling. To make it properly better to use pseudo-elements but not for this time.